test/gpdb/scripts/test_pitr_gpdb6.sh

```diff
@@ -0,0 +1,201 @@
+        "${current_date}-??????F" -not -empty ) ]];
+    then
+        echo "Found a backup directory for segment $1: $dirname" > /dev/null
+    else
+        echo "The backup directory for segment $1 was not found" > /dev/null
+        exit 1
+    fi
+}
+for i in -1 0
+do 
+   check_backup $i
+done
+
+echo "Creating additional table..." > /dev/null
+psql -d gpdb_pitr_database -c \
+"CREATE TABLE t2 (id int, text varchar(255)) DISTRIBUTED BY (id);"
+
+psql -d gpdb_pitr_database -c \
+"INSERT INTO t2 SELECT i, 'text'||i FROM generate_series(1,30) i;"
+
+psql -d gpdb_pitr_database -c "SELECT * FROM t2 ORDER BY id;" \
+-o $PGBACKREST_TEST_DIR/$TEST_NAME/t2_rows_original.out
+
+echo "Creating full backup on seg1 and seg2..." > /dev/null
+for i in 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
+    --config $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i \
+    --type=full backup
+done
+
+echo "Checking the presence of second backup" > /dev/null
+for i in 1 2
+do 
+   check_backup $i
+done
+
+echo "Creating a distributed restore point" > /dev/null
+psql -d gpdb_pitr_database -c "create extension gp_pitr;"
+psql -d gpdb_pitr_database -c "select gp_create_restore_point('test_pitr');"
+psql -d gpdb_pitr_database -c "select gp_switch_wal();"
+
+echo "Simulating disaster..." > /dev/null
+psql -d gpdb_pitr_database -c "drop table t1;"
+psql -d gpdb_pitr_database -c "truncate table t2;"
+
+gpstop -a
+rm -rf $MASTER/* $PRIMARY1/* $PRIMARY2/* $PRIMARY3/*
+rm -rf $MIRROR1/* $MIRROR2/* $MIRROR3/* $DATADIR/standby/*
+
+echo "Restoring cluster..." > /dev/null
+for i in -1 0 1 2
+do 
+	PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest --config \
+    $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i --type=name \
+    --target=test_pitr restore
+done
+
+echo "Configure mirrors after primary restore" > /dev/null
+gpstart -am
+gpinitstandby -ar
+
+PGOPTIONS="-c gp_session_role=utility" psql -d gpdb_pitr_database << EOF
+SET allow_system_table_mods to true;
+UPDATE gp_segment_configuration
+SET status = CASE WHEN role='m' THEN 'd' ELSE status END, mode = 'n'
+WHERE content >= 0;
+EOF
+
+gpstop -ar
+gprecoverseg -aF
+gpinitstandby -as $HOSTNAME -S $DATADIR/standby/ -P 6001
+
+echo "Checking data integrity" > /dev/null
+psql -d gpdb_pitr_database -c "SELECT * FROM t1 ORDER BY id;" \
+-o $PGBACKREST_TEST_DIR/$TEST_NAME/t1_rows_restored.out
+
+psql -d gpdb_pitr_database -c "SELECT * FROM t2 ORDER BY id;" \
+-o $PGBACKREST_TEST_DIR/$TEST_NAME/t2_rows_restored.out
+
+if diff "$PGBACKREST_TEST_DIR/$TEST_NAME/t1_rows_original.out" \
+"$PGBACKREST_TEST_DIR/$TEST_NAME/t1_rows_restored.out"; then
+    echo "Rows match." > /dev/null
+else
+    echo "Discrepancy in rows found." > /dev/null
+    exit 1
+fi
+
+if diff "$PGBACKREST_TEST_DIR/$TEST_NAME/t2_rows_original.out" \
+"$PGBACKREST_TEST_DIR/$TEST_NAME/t2_rows_restored.out"; then
+    echo "Rows match." > /dev/null
+else
+    echo "Discrepancy in rows found." > /dev/null
```

Comment:
the echo is ending somewhere `...` somewhere without. and somewhere like here `.`. does it have any meaning? Maybe it should be done uniformly everywhere?